qa/tasks/ceph_manager.py

```diff
@@ -1671,14 +1674,14 @@ def expand_pool(self, pool_name, by, max_pgs):
             self.pools[pool_name] = new_pg_num
             return True
 
-    def set_pool_pgpnum(self, pool_name):
+    def set_pool_pgpnum(self, pool_name, force):
         """
         Set pgpnum property of pool_name pool.
         """
         with self.lock:
             assert isinstance(pool_name, basestring)
             assert pool_name in self.pools
-            if self.get_num_creating() > 0:
```

Comment:
why does this expect to see creating pgs when it finishes anyway?  maybe it should set pgp_num unconditionally, but block in a loop first waiting for creating pgs to get created?